src/activities/CategoriesEntity.js

```diff
@@ -89,6 +89,29 @@ export class CategoriesEntity extends Entity {
 		return Number(category.categoryId) === Number(selectedCategory.properties.categoryId);
 	}
 
+	getSelectedCategoryHref(categoryId) {
+		if (!this._entity) {
+			return;
+		}
+
+		const subEntities = this._entity.getSubEntitiesByClass(Classes.activities.category);
+
+		let selectedCategory;
+		if (subEntities) {
+			selectedCategory = subEntities.find(category => Number(category.properties.categoryId) === categoryId);
```

Comment:
TODO if we have time later - do the number check here. I know you handled it in activities so this works but my bad for not maintaining consistency here 😅

Reply:
Added the cast